packages/rule-utils/src/themeFn.ts

```diff
@@ -4,7 +4,9 @@ import { colorToString, parseCssColor } from './colors'
 export const themeFnRE = /theme\(\s*['"]?(.*?)['"]?\s*\)/g
 
 export function hasThemeFn(str: string) {
-  return str.includes('theme(') && str.includes(')')
```

Comment:
Personally, I think this is enough. We don't need completely accurate results here. This may also be a small performance optimization @zyyv

Reply:
I hold another opinion. I think strict judgment should be maintained here. Performance optimization here is ignorable.

Reply:
Yeah I agree with @Dunqing that `String.includes` is enough to serve as the early return guard. We could let `transformThemeFn` return `false` if not matched. This way we keep the common utils performant, while keeping it accurate.